pkg/cmd/pr/shared/templates.go

```diff
@@ -129,14 +129,14 @@ type templateManager struct {
 	fetchError error
 }
 
-func NewTemplateManager(httpClient *http.Client, repo ghrepo.Interface, dir string, allowFS bool, isPR bool) *templateManager {
+func NewTemplateManager(httpClient *http.Client, cachedClient *http.Client, repo ghrepo.Interface, dir string, allowFS bool, isPR bool) *templateManager {
```

Comment:
This was annoying having to change this function definition to include a cached client, but I couldn't come up with a less intrusive way to make sure the feature detector got a cached client.